frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/molecules/CustomDropdown.js

```diff
@@ -71,7 +71,7 @@ const CustomDropdown = ({ t, config, inputRef, label, onChange, value, errorStyl
     {
       select: (data) => {
         const optionsData = _.get(data, `${config?.mdmsConfig?.moduleName}.${config?.mdmsConfig?.masterName}`, []);
-        return optionsData.filter((opt) => opt?.active).map((opt) => ({ ...opt, name: `${config?.mdmsConfig?.localePrefix}_${Digit.Utils.locale.getTransformedLocale(opt.code)}`, code: `${config?.mdmsConfig?.localePrefix}_${Digit.Utils.locale.getTransformedLocale(opt.code)}` }));
+        return optionsData.filter((opt) => opt?.active).map((opt) => ({ ...opt, name: `${config?.mdmsConfig?.localePrefix}_${Digit.Utils.locale.getTransformedLocale(opt.code)}` }));
```

Comment:
@nipunarora-eGov @hariomsinha-eGov @prachirane-eGov  please check if we remove code it wont impact other places